build-android/app/src/main/java/com/google/firebase/codelab/friendlychat/ButtonObserver.kt

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.firebase.codelab.friendlychat
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.Button
+
+class ButtonObserver(private val mButton: Button) : TextWatcher {
```

Comment:
The reason I use the `My` prefix on these classes is to make sure that the person doing the codelab doesn't think that `ButtonObserver` is a standard Android class, they know it's something custom. It's definitely ugly to use a useless prefix but I think it helps. What do you think about that? We could use something like `Friendly` instead of `My`.

(we could also use no prefix, I don't feel strongly about this)

Reply:
I didn't think of it like that. I think you're right, using the "My" prefix is a good indicator that it's not a standard class. I'll go back to using "My" 👍